aidants_connect_web/models.py

```diff
@@ -110,7 +110,16 @@ def active(self):
 class Aidant(AbstractUser):
     profession = models.TextField(blank=False)
     organisation = models.ForeignKey(
-        Organisation, null=True, on_delete=models.CASCADE, related_name="aidants"
+        Organisation,
+        null=True,
+        on_delete=models.CASCADE,
+        related_name="current_aidants",
+    )
+    organisations = models.ManyToManyField(
+        Organisation,
+        related_name="aidants",
+        blank=True,
+        verbose_name="Organisation active",
```

Comment:
Du coup ici le `verbose_name` ce serait plutôt "Organisations possibles" ?

Reply:
```suggestion
        verbose_name="Organisations possibles",
```

Pas complètement fan du nommage, je pense qu'on pourrait faire plus clair… Mais c'est purement interne AC donc on pourra s'en réoccuper plus tard.